modules/gfaffix/main.nf

```diff
@@ -0,0 +1,33 @@
+process GFAFFIX {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? 'bioconda::gfaffix=0.1.2.3' : null)
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gfaffix:0.1.2.3--h779adbc_0' :
```

Comment:
The most recent version is 0.1.4 https://anaconda.org/bioconda/gfaffix.

Reply:
So before merging I would suggest to update.

Reply:
I would agree, would be good to bump the version before we merge.

Reply:
Resolved by commit fdeeb2bc6a4f79bb6fc69a7d4e4554f04802753c